src/compositions/Timeline/Timeline.tsx

```diff
@@ -7,20 +7,12 @@ import "./Timeline.css";
 
 const Timeline = (props: any) => {
   const { statusConfig, unitTestSuccess, hasFailedTasks } = props;
-  const { buildInfo } = useAppSelector((state) => state.runTime);
 
   return (
     <div className="statusTimeline" data-testid="statusTimeline">
       <ul className="statusTimelineUl">
-        {statusConfig.map(
-          (config: any, index: React.Key | null | undefined) => {
-            if (buildInfo.runState === config.status) {
-              config.runTimeTaken = buildInfo.runTime
-            }
```

Comment:
Why is this removed? 
We have to show the run time taken under each of the timeline phase once it is completed.

Reply:
Ok, is fixed now